.travis.yml

```diff
@@ -1,9 +1,12 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
   - "8"
 
+os:
+  - linux
+  - osx
```

Comment:
Can we leave `osx` support out of this patch please?

Reply:
Yeah I was in the process of removing it :)